ansible/templates/usb/scripts/headnode.sh.j2

```diff
@@ -226,6 +226,7 @@ function _build_vmmanifest() {
 				-e "s|@REDIS_PASSWORD@|${REDIS_PASSWORD}|" \
 				-e "s|@PGSQL_ESDC_PASSWORD@|${PGSQL_ESDC_PASSWORD}|" \
 				-e "s|@PGSQL_PDNS_PASSWORD@|${PGSQL_PDNS_PASSWORD}|" \
+				-e "s|@PGSQL_MGMT_MON_PASSWORD@|${PGSQL_MGMT_MON_PASSWORD}|" \
```

Comment:
Is this necessary PGSQL_MGMT_MON_PASSWORD? I would prefer that the `es-post-deploy.sh` script would do the job...

Reply:
We need to be aware of the password for other purposes